### ISSUE TITLE
Joint use Leaflet.draw and Leaflet.markercluster

### ISSUE THREAD

**Issue by absemetov** [Leaflet/Leaflet.draw#171]
When cluster reopen, draw crash

``` javascript
  var editableLayers = new L.MarkerClusterGroup({maxClusterRadius: 50});

  map.addLayer(editableLayers); 

  var drawControl = new L.Control.Draw({
    draw: {
      position: 'topleft',
      polyline: false,
      polygon: false,
      rectangle: false,
      circle: false,
      marker: {
        repeatMode: true
      }
    },
    edit: {
      featureGroup: editableLayers
    },

  });

  map.addControl(drawControl);
```

**Comment by mfr57** [comment]
Please update, are "draw" and "clusters" still incompatible?

**Comment by danzel** [comment]
You will never be able to use Leaflet.Draw on a Leaflet.MarkerClusterGroup layer.

**Comment by mfr57** [comment]
Well I refuse to accept that answer, so if I find a way to prove you wrong I will be sure to post :) Until then, thank you anyways.

**Comment by danzel** [comment]
Most people use another layer for editing, then transfer the markers to the cluster layer when editing is done.

We use a separate layer for adding markers, then when toggling moving a marker we remove it from the MCG and put it on the map to move it. Markers can be directly removed from a MCG.

**Comment by rzeznik-kasia** [comment]
@mfr57 - did you succeeded?

**Comment by mfr57** [comment]
@rzeznik-kasia No, I instead added the clusters then using the Leaflet API, I added the individual editing tools to the map. I created a custom modal form that would collect information and used that to add new attributes, then added code to set it up so when the user clicks on the map it adds the geometry. It was roundabout but I ended up letting people "add" (I didn't need them to be able to update or delete) while still using clusters for my markers. This is old now though, so they may have updated Leaflet so that they are compatible OOTB.
